code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -347,8 +347,10 @@
 			var/dmg = rand(M.force/2, M.force)
 			switch(M.damtype)
 				if("brute")
-					if(M.force > 20)
+					if(M.force > 35) // durand and other heavy mechas
 						Unconscious(20)
+					else if(M.force > 20 && !IsKnockdown()) // lightweight mechas like gygax

```

Comment:
can you avoid mechapunch if you rest lol?

Reply:
yes you avoid being knocked down if you crawl (the damge is same)

Reply:
tactical resting is still alive lmfao